autotest/config.yaml

```diff
@@ -370,3 +373,17 @@ benchmark_model:
     - deepseek-ai/DeepSeek-V2-Lite-Chat
     - openai/gpt-oss-20b
     - openai/gpt-oss-120b
+
+
+evaluate_model:
+  - google/gemma-2-9b-it
```

Comment:
Please remove the following models:
- internlm/internlm3-8b-instruct
- meta-llama/Llama-2-7b-chat-hf
- internlm/internlm2_5-20b-chat